### ISSUE TITLE
Assignment without usage.

### ISSUE THREAD

**Issue by PhilipMay** [richarddwang/electra_pytorch#18]
Hi,

here you assign something but do not use it later.
Smells like a small bug.

https://github.com/richarddwang/electra_pytorch/blob/208c62fc3220b3ba318b5c9aed87d4ec75a3bea2/pretrain.py#L377

**Comment by richarddwang** [comment]
Thanks for the report
Yes it is a bug.
Although it doesn't affect default learning rate schedule that is the same as the official one uses, I have committed a change to pretrain.py.

Thanks you !
